src/main/java/entities/Company.java

```diff
@@ -0,0 +1,26 @@
+package entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j;
+
+import javax.persistence.*;
```

Comment:
Please don't use a wild card in imports.
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad